ecl/hql/hqlfold.cpp

```diff
@@ -5582,6 +5590,14 @@ IHqlExpression * CExprFolderTransformer::createTransformed(IHqlExpression * expr
     node_operator op = expr->getOperator();
     switch (op)
     {
+    case no_likely:
+    case no_unlikely:
```

Comment:
This code isn't needed - it should be handled by the code above.  No need to duplicate.